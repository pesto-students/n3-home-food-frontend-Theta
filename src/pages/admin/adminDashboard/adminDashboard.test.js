import * as React from "react";
import renderer from "react-test-renderer";
import  AdminDashboard from './adminDashboard'

describe('snapshot testing admin dashboard component', ()=> {
   
    it('line cart component',()=> {
        const tree = renderer.create(<AdminDashboard />).toJSON();
        expect(tree).toMatchSnapshot();
    } )
} )

