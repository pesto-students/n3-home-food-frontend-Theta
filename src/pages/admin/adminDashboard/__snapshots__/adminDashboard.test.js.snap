// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing admin dashboard component line cart component 1`] = `
<p>
  Admin Dashboad
</p>
`;
